apps/web/src/components/templates/email-editor/EmailContentCard.tsx

```diff
@@ -29,7 +32,17 @@ export function EmailContentCard({
     watch,
   } = useFormContext(); // retrieve all hook methods
   const contentType = watch(`steps.${index}.template.contentType`);
+  const subject = watch(`steps.${index}.template.subject`);
   const [activeTab, setActiveTab] = useState(0);
+  const { integrations = [] } = useIntegrations();
```

Comment:
this hooks is only returning active integrations so there should only be one email integration